cinema/views.py

```diff
@@ -1,45 +1,123 @@
-from rest_framework.decorators import api_view
+from rest_framework.generics import GenericAPIView
+from rest_framework.mixins import (
+    ListModelMixin,
+    CreateModelMixin,
+    RetrieveModelMixin,
+    UpdateModelMixin,
+    DestroyModelMixin
+)
+from rest_framework.request import Request
 from rest_framework.response import Response
 from rest_framework import status
 
 from django.shortcuts import get_object_or_404
+from rest_framework.views import APIView
+from rest_framework.viewsets import GenericViewSet, ModelViewSet
```

Comment:
same as above (see other similar cases) and (better) use autoformatter (black etc.)